mico-admin/src/app/api/api.service.ts

```diff
@@ -427,23 +427,35 @@ export class ApiService {
 
     }
 
-    postServiceViaGithub(data): Observable<Readonly<ApiObject>> {
-        if (data == null) {
-            return;
-        }
+    getServiceVersionsViaGithub(uri: string): Observable<Readonly<ApiObject[]>> {
```

Comment:
type should be string[]

Reply:
respectfully disagree. The uri is the github link to the repository where the version numbers shall be crawled. Hence we need only one string. However, I will add a comment to clarify this.

Reply:
Clarification: return type should be `Observable<Readonly<string[]>>` (like in the linked issue)